netbox/extras/forms/model_forms.py

```diff
@@ -531,6 +531,15 @@ def __init__(self, *args, **kwargs):
                 if type(value) in (tuple, list):
                     value = ', '.join(value)
                 self.fields[param.name].initial = value
+
+                if param.name == 'CUSTOM_VALIDATORS':
```

Comment:
Rather than focusing on `CUSTOM_VALIDATORS` specifically, this would be a good check to implement for _all_ parameters.